MailSender/src/main/java/com/metacube/MailSender/service/MailSender.java

```diff
@@ -0,0 +1,14 @@
+package com.metacube.MailSender.service;
+/**
+ * 
+ * @author Abhishek Bishnoi
+ * @since  Sept 4, 2019
+ * This is interface of Mailsender
+ */
+public interface MailSender {
+	/**
+	 * This function writes the message from where we send a mail
+	 * @return message String
+	 */
+	String sendMail();
```

Comment:
For sending mail, we need few basic parameters as message, recipients, subject. All these can be the arguments of sendMail method.